src/System.Collections.Concurrent/ref/System.Collections.Concurrent.csproj

```diff
@@ -2,9 +2,10 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.10.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(TargetGroup)'=='netstandard1.3'">4.0.10.0</AssemblyVersion>
```

Comment:
As I said on my previous comment on the dir.props file, I misguided you to update this AssemblyVersion, since it looks like we only want to update the AssemblyVersion whenever we expose new API for `netstandard` and not when we only expose it for `netcoreapp`. That means that this should go back to being: `<AssemblyVersion>4.0.10.0</AssemblyVersion>` That should fix your build issue.